custom_model_runner/datarobot_drum/drum/utils/structured_input_read_utils.py

```diff
@@ -84,8 +84,13 @@ def read_structured_input_data_as_df(binary_data, mimetype, sparse_colnames=None
 
                 return df
             else:  # CSV format
-                df = pd.read_csv(io.BytesIO(binary_data))
-
+                try:
+                    df = pd.read_csv(io.BytesIO(binary_data))
+                except UnicodeDecodeError:
+                    logger.error(
+                        "A non UTF-8 encoding was encountered while opening the data.\nSave this using utf-8 encoding, for example with pandas to_csv('filename.csv', encoding='utf-8')."
+                    )
+                    raise DrumCommonException("Supplied CVS input must utilize UTF-8 encoding.  ")
```

Comment:
typo and suggestion:
```suggestion
                    raise DrumCommonException("Supplied CSV input file encoding must be UTF-8.")
```